README.md

```diff
@@ -1 +1,4 @@
-# apple-clone-review
\ No newline at end of file
+# Training Purpose
+
+### This is a Mint sign up form copied from my past project.
+
```

Comment:
- [ ] please add descriptive readme, you can use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project